test/test.cpp

```diff
@@ -43,6 +43,15 @@ int main() {
 
     assert(!orm::exists<Person>(conn));
 
+    Person person{
+        .id = 42,
+        .name = "Alex",
+    };
+
+    orm::persist(conn, person);
```

Comment:
Why make it a function and not a Connection method?
```suggestion
    conn.persist(person);
```

Reply:
I am planning to do it later after i finish prototyping